src/js/utils/dom-utils.js

```diff
@@ -92,6 +92,25 @@ function parseHTML(html) {
   return div;
 }
 
+function closest(el, selector, contentOnly=false) {
+  if (!isElementNode(el)) {
+    el = el.parentElement;
```

Comment:
This needs to be `el = el.parentNode` in order for IE to pass. IE doesn't define `parentElement` on `Node`, only on `Element` (and a textNode is a `Node` but not an `Element`). (Thanks a lot, IE.)